tests/layer_tests/onnx_tests/test_conv.py

```diff
@@ -477,11 +477,12 @@ def test_conv_4D_autopad(self, params, auto_pad, bias, ie_device, precision, ir_
     @pytest.mark.precommit
     def test_conv_5D_precommit(self, params, dilations, pads, strides, bias, ie_device, precision,
                                ir_version, temp_dir, use_old_api):
+        custom_eps_value = 1e-1 if ie_device == 'GPU' and precision == 'FP16' else None
```

Comment:
@rkazants custom threshold  already sets for only GPU && FP16

Reply:
What does `None` value mean for `custom_eps_value`. Now you seem to have changed this behavior for other plugins.

Reply:
Looks good - my bad:)

Reply:
Nice, thanks!